### PR TITLE
Fix #6265

### LINKED ISSUE #6265: Supermatter shards vaporize other shards
wah

### PR DESCRIPTION
**Pull request by clusterfack**
Humorous flavor text and they explode, I think that satifies everyone's wishes for it. Personally I couldn't care less, what are the chances of this even happening if someone isn't looking to do it intentinoally.

### CODE REVIEW

File: code/modules/supermatter/supermatter.dm [hunk 1]
```diff
@@ -381,6 +381,12 @@
 
 
 /obj/machinery/power/supermatter/Bumped(atom/AM as mob|obj)
+	if(istype(AM, /obj/machinery/power/supermatter))
+		AM.visible_message("<span class='danger'>As the [src] bumps into the [AM] an otherworldly resonance ringing begins to shake the room, you ponder for a moment all the incorrect choices in your life that led you here, to this very moment, to witness this. You take one final sigh before it all ends.</span>")
```
Comment (ghost) [inline code/modules/supermatter/supermatter.dm]: \the

**Comment by 9600bauds** [comment]
Probably make this sinister span

**Comment by dylanstrategie** [comment]
For some reason I imagined that an event like this wouldn't make the usual supermatter sound, but instead https://www.youtube.com/watch?v=S4llk8F7vLs

Yes, even with the "NVIDIA" part, we need funding afterall

**Comment by dylanstrategie** [comment]
This would need a 5 seconds sleep, but then the explosion is pretty strong and you can likely add something fancy like flashing a glow animation

**Comment by gbasood** [comment]
Since it's a bugfix I think that any QUALITY soundeffects or visual effects can be done by anyone who wants to do them later.

**Pull request merged**